### ISSUE TITLE
Crash on 1.18.2

### ISSUE THREAD

**Issue by enderprism** [shedaniel/slight-gui-modifications#63]
My game crashes with this mod on 1.18.2.

**Comment by ultra-black** [comment]
How in the world is anyone supposed to know the issue without logs.

**Comment by muhammedesadcomert** [comment]
My game also crashes on 1.18.2

#### Versions:
* Fabric: 0.13.3
* Mod: slight-gui-modifications-2.1.0

Crash logs: https://pste.ch/uzijopupic.sql
Logs: https://pste.ch/zakabigeli.cs

**Comment by Madis0** [comment]
Yup, this is probably because ARRP is listed as a required dependency while it is actually already included in SGM, plus https://github.com/Devan-Kerman/ARRP/issues/60
![image](https://user-images.githubusercontent.com/8611110/157080367-624268f6-6e86-4519-977c-521dbbbd19b7.png)

**TL;DR: remove ARRP jar from your mods folder**

**Comment by shedaniel** [comment]
ARRP has been updated

**Comment by muhammedesadcomert** [comment]
Thanks, update fixed the problem, issue can be closed
